modules/apps/frontend-data-set/frontend-data-set-impl/src/main/resources/com/liferay/frontend/data/set/internal/batch/01-object-definition.batch-engine-data.json

```diff
@@ -20,6 +20,22 @@
 			"modifiable": true,
 			"name": "DataSet",
 			"objectFields": [
+				{
+					"DBType": "Boolean",
+					"businessType": "Boolean",
+					"externalReferenceCode": "INACTIVE",
+					"indexed": true,
+					"indexedAsKeyword": false,
+					"label": {
+						"en_US": "Inactive"
+					},
+					"localized": false,
+					"name": "inactive",
```

Comment:
> We have found that it is not possible to set a defaultValue for Boolean fields, so when we apply this change all existing Data Sets and Data Sets sub-entities will have a new property with a false value. This is the main reason to use inactive instead of active.

I think this is fine. It's a bit unusual to name with a negative, but it's not unheard of. `disabled` for example is pretty common in UI. The logic is that flag "on state" represents the unusual, less common state.